lib/db/mysql.js

```diff
@@ -701,6 +701,35 @@ module.exports = function (log, error) {
     return this.write(UPDATE_LOCALE, [data.locale, uid])
   }
 
+  var CREATE_UNBLOCK_CODE = 'CALL createUnblockCode_0(?, ?, ?)'
+
+  MySql.prototype.createUnblockCode = function (uid, code) {
+    return this.write(
+      CREATE_UNBLOCK_CODE,
+      [ uid, code, Date.now() ],
```

Comment:
There are probably some things in here we could handle a little more carefully, although hawk tokens are a special case, because the 'id' portion of the token doesn't need to be kept secret.